lab-matt/test/model-route-test.js

```diff
@@ -0,0 +1,132 @@
+'use strict';
```

Comment:
You've tested your code well.  Great job!  There was only one test missing, you'll need to add a test which checks to make sure a 404 is returned if a route which doesn't exist is specified (in other words a catch all route), meaning one to make sure that if a client requests /api/isodnghosbgn it returns a 404 error.